client/src/app/app.routes.ts

```diff
@@ -11,7 +11,7 @@ import { ProvincialRecordsComponent } from './pages/provincial-records/provincia
 import { PrivacyComponent } from './pages/privacy/privacy.component';
 import { TermsComponent } from './pages/terms/terms.component';
 import { VolunteerComponent } from './pages/volunteer/volunteer.component';
-import { faqsComponent } from './pages/faqs/faqs.component';
+import { FAQComponent } from './pages/FAQ/FAQ.component';
```

Comment:
The naming conventions (which to be fair isn't well documented) is `PascalCaseComponent` for class names, and `kebab-case.component` for the filenames and paths. So this line should read:

```
import { FAQComponent } from './pages/faq/faq.component';
```

Reply:
Done.

Reply:
This issue is still there and it's also the cause of the failing test. You need to rename the `faq.component.ts` file and the `faq` folder to lower case, but you may be running into issues with Windows file handling. If you try to rename a folder by changing the case only, Windows will change how the folder is displayed but won't actually change the case. The test environment and the production server are both Linux, so they won't work until this is resolved. The easiest way to change the folder names is to rename the file and the folder to something else (such as `pages/faq_renaming/faq_renaming.component.ts`) being careful to use all lower case, commit that, and then rename it to `pages/faq/faq.component`. Then make sure you get all of the cases in the imports correct, because this will work on Windows but fail the CI tests and in production.